src/custom-webpack/index.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (context, options) {
+  return {
+    name: 'custom-webpack',
+    configureWebpack(config, isServer, utils) {
+      return {
+        optimization: {
+          splitChunks: {
+            cacheGroups: {
+              common: {
+                minChunks: 1
```

Comment:
Can you check whether a value like `5` or `10` may lead to a better experience than moving everything into common?